data/actionsmap/yunohost.yml

```diff
@@ -291,6 +291,17 @@ domain:
                     extra:
                         pattern: *pattern_domain
 
+        ### domain_generate_dns_config()
+        generate_dns_configuration:
+            action_help: Generate DNS configuration for a domain
+            api: GET /domains/<domain>/dns_config
```

Comment:
Maybe it's just a personnal taste, but I would prefer a dash instead of an underscore in url. or maybe just `/domain/<domain>/dns`.